specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/applicationGateway.json

```diff
@@ -40,6 +40,11 @@
           "ApplicationGateways"
         ],
         "operationId": "ApplicationGateways_Delete",
+        "x-ms-examples": {
+          "Delete ApplicationGateway": {
```

Comment:
nit: Spaces are not recommended as json keys,
consider naming this as `Delete ApplicationGateway`
This applies to all examples here

Reply:
Is this a new change? Every example name we have across all our specs have spaces in them.

Reply:
Not a new change, but moving forward it would be nice :) won't block on this

Reply:
ok, thanks. I'll look at doing this in April or May version. Thanks!